docs/resources/CHANNEL.md

```diff
@@ -228,6 +228,9 @@ Returns the messages for a channel. If operating on a guild channel, this endpoi
 
 Returns a specific message in the channel. If operating on a guild channel, this endpoints requires the 'READ_MESSAGE_HISTORY' permission to be present on the current user. Returns a [message](#DOCS_CHANNEL/message-object) object on success.
 
+>warn
+> Only bot accounts can use this endpoint.
```

Comment:
it's hard to distinguish is "this" applies to the following endpoint or the above endpoint